.travis.yml

```diff
@@ -43,11 +41,12 @@ before_script:
   - for i in {1..4}; do echo "$(sleep 500)" ; done &
   - set -o errexit; source ./ci/test/05_before_script.sh &> "/dev/null"
 script:
+  - if [[ -n "$USE_VALGRIND" || "$RUN_FUNCTIONAL_TESTS" = "true" || "$RUN_FUZZ_TESTS" = "true" ]]; then export LONG_RUNNING_TEST=1; fi
```

Comment:
are the fuzz tests timing out as well these days?

Reply:
I hope not! Do we have any evidence suggesting that? If so I'll investigate how we could speed things up.